shifthelper/conditions.py

```diff
@@ -345,7 +350,10 @@ def is_nobody_on_shift():
 @log_call_and_result
 def is_last_shutdown_already_10min_past():
     '''Last Shutdown is already 10min past'''
-    return get_last_shutdown() + timedelta(minutes=10) < datetime.utcnow()
+    return (
+        get_last_shutdown() + timedelta(minutes=10) + TIME_BETWEEN_CLONES <
```

Comment:
I'm really not sure about this one. Maybe drop this and just clone the parking checklist more often? Like once a minute?

Reply:
I wonder if cloning more often defeats the purpose of cloning...

Reply:
but maybe your are right